cast/src/lib.rs

```diff
@@ -1077,6 +1080,85 @@ impl SimpleCast {
         }?)
     }
 
+    /// Encodes a string or list to hexadecimal rlp
+    ///
+    /// ```
+    /// use cast::SimpleCast as Cast;
+    ///
+    /// fn main() -> eyre::Result<()> {
+    ///    assert_eq!(Cast::to_rlp("\"foundry\"".to_string()).unwrap(), "0x87666f756e647279" );
+    ///    assert_eq!(Cast::to_rlp("[]".to_string()).unwrap(), "0xc0");
+    ///    assert_eq!(Cast::to_rlp("[\"a\"]".to_string()).unwrap(), "0xc161");
+    ///    assert_eq!(Cast::to_rlp("[\"a\",\"b\"]".to_string()).unwrap(), "0xc26162");
+    ///     Ok(())
+    /// }
+    /// ```
+    pub fn to_rlp(value: String) -> Result<String> {
+        let val = serde_json::from_str(&value)?;
+        let item = rlp_converter::value_to_item(&val, false);
+        Ok(format!("0x{}", hex::encode(rlp::encode(&item))))
+    }
+
+    /// Decodes rlp encoded hexadecimal string
+    ///
+    /// ```
+    /// use cast::SimpleCast as Cast;
+    ///
+    /// fn main() -> eyre::Result<()> {
+    ///     assert_eq!(Cast::from_rlp("0x87666f756e647279".to_string()).unwrap(), "\"foundry\"");
+    ///     assert_eq!(Cast::from_rlp("87666f756e647279".to_string()).unwrap(), "\"foundry\"");
+    ///     assert_eq!(Cast::from_rlp("0xc0".to_string()).unwrap(), "[]");
+    ///     assert_eq!(Cast::from_rlp("c161".to_string()).unwrap(), "[\"a\"]");
+    ///     assert_eq!(Cast::from_rlp("0xc26162".to_string()).unwrap(), "[\"a\",\"b\"]");
+    ///     Ok(())
+    /// }
+    /// ```
+    pub fn from_rlp(value: String) -> Result<String> {
+        let striped_value = strip_0x(&value);
+        let bytes = hex::decode(striped_value).expect("could not decode hex");
```

Comment:
Since these errors are user facing I think we should try to make them a bit more verbose/"prettier"

```suggestion
        let bytes = hex::decode(striped_value).expect("Could not decode hex");
```